the-collab-lab/tcl-38-smart-shopping-list
# install tailwind, update react-scripts

## Description

This PR installs the TailwindCSS package to prepare for styling the app. 

## Related Issue
Closes #39. 

## Acceptance Criteria

Install TailwindCSS package in the project  by following the [Tailwind Installation Documentation](https://tailwindcss.com/docs/guides/create-react-app).

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|   | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
| ✓   | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

| Before | After |
|---|---|
| ![image](https://user-images.githubusercontent.com/40015502/155247580-a3771048-e6a6-4584-9a7f-50587517bd71.png) | ![image](https://user-images.githubusercontent.com/40015502/155247595-f1a2927d-20cf-4af3-8735-5686e295926e.png) |


## Testing Steps / QA Criteria

Pull down the branch and run npm install. Restart your server, and try changing some styling such as:`<p className="text-blue-500 text-2xl">Join an existing shopping list by entering a three word token</p>` inside of the `Home.js` component. Text color for "Join an existing shopping list..." should change to blue if Tailwind is installed correctly.
